packages/server/__snapshots__/7_record_spec.coffee.js

```diff
@@ -766,21 +766,19 @@ Can't run because you've entered an invalid browser name.
 Browser: 'browserDoesNotExist' was not found on your system or is not supported by Cypress.
 
 Cypress supports the following browsers:
-- Canary
-- Chrome
-- Chromium
-- Edge
-- Edge Beta
-- Edge Canary
-- Edge Dev
-- Electron
-- Firefox (Beta support)
-- Firefox Developer Edition (Beta support)
-- Firefox Nightly (Beta support)
+- chrome
+- chromium
+- edge
+- electron
+- firefox (Cypress support in beta)
 
 You can also use a custom browser: https://on.cypress.io/launching-browsers#Customize-available-browsers
 
-Available browsers found are: browser1, browser2, browser3
+Available browsers found are:
+- chrome
+- firefox
+- edge
+- electron
```

Comment:
you'll also need to update the availableBrowsersRe in e2e.js so it can sanitize the snapshots for this correctly:

https://github.com/cypress-io/cypress/blob/a3bed447c5f99e3dc9be3c90fef45cdf412f35af/packages/server/test/support/helpers/e2e.js#L43-L43

the current regex just replaces the entire line with browser1, browser2, browser3, but since it's multiline now you'll need a new regex

https://github.com/cypress-io/cypress/blob/a3bed447c5f99e3dc9be3c90fef45cdf412f35af/packages/server/test/support/helpers/e2e.js#L137-L144